.environment-scripts/post_build_steps.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+FV3NET_DIR=$1
+
+# Make microphysics scripts executable
+chmod +x $FV3NET_DIR/projects/microphysics/scripts/*
```

Comment:
I think alternatively we could commit these scripts as executable, but maybe there is a reason we do not do that already.

Reply:
I agree that's a better solution.

Reply:
Sounds good.  I went ahead and did that.